### PR TITLE
added tests for filters on search page

### PR DESCRIPTION
**Pull request by pbadak**

### CODE REVIEW

File: suites/test_search_and_filter.py [hunk 1]
```diff
@@ -77,3 +78,25 @@ def search_for_body_of_module(self):
...
+        utilities.wait(2)
+        utilities.click_element(self.driver, By.XPATH, locators.SEARCH_MODULE_XPATH)
+        utilities.wait(2)
+        check_that("verify that 'filter by product and version' functionality filters results correctly",
+                   utilities.get_text(self.driver, By.XPATH, locators.PRODUCT_VERSION_DISPLAY_PAGE_XPATH),
+                   contains_string("AT Product Test 1"))
+
+    @lcc.test("Verify that 'Filter by content Type': Assembly functionality filters results correctly")
+    def select_content_type_filter(self):
+        verify_filter_by_module_type(self.driver, "Assembly")
```
Comment (noopurmath3) [inline suites/test_search_and_filter.py:102]: @pbadak this can be renamed to verify_filter_by_content_type() now, to be generic

**Review by noopurmath3 — APPROVED** [review]
Added a comment
Looks good to me :+1:

File: suites/test_search_and_filter.py (hunk 1 above)
Comment (pbadak) [reply suites/test_search_and_filter.py]: changes incorporated.

**Review by anarang — APPROVED** [review]
looks good, thanks Pournima. :)

**Pull request merged**